airdrop/project/contracts/distributor-contract/src/main.sw

```diff
@@ -82,8 +132,17 @@ impl Info for Contract {
     }
 
     #[storage(read)]
-    fn merkle_root() -> b256 {
-        require(storage.merkle_root.is_some(), InitError::NotInitalized);
-        storage.merkle_root.unwrap()
+    fn is_active() -> bool {
+        storage.end_block > height()
+    }
+
+    #[storage(read)]
+    fn merkle_root() -> Option<b256> {
```

Comment:
Under what conditions would there not be a merkle_root?

Reply:
Before the contract has been initialized.